gateway/1.5/modules/ROOT/pages/index.adoc

```diff
@@ -7,6 +7,12 @@ endif::[]
 
 Anypoint Flex Gateway is an Envoy-based, ultrafast lightweight API gateway designed to manage and secure APIs running anywhere. Built to seamlessly integrate with DevOps and CI/CD workflows, Flex Gateway delivers the performance required for the most demanding applications and microservices while providing enterprise security and manageability across any environment.
 
+Watch the Flex Gateway Product Spotlight video to see a quick overview of Flex Gateway.
```

Comment:
I think we're trying to not use words like "watch" and "see", as they're not inclusive in terms of accessibility. I know this language is used to introduce videos in other product pages, but per our team meeting today you might want to experiment with something different.

Maybe:
"The following video provides a quick overview of Flex Gateway:"

Reply:
Yeah, I like that better. I just used what Beth had in her example, but I agree with this.